RecoHGCal/Configuration/python/RecoHGCal_EventContent_cff.py

```diff
@@ -70,3 +99,19 @@ def cleanOutputAndSet(outputModule, ticl_outputCommads):
         cleanOutputAndSet(process.FEVTDEBUGHLToutput, TICL_FEVT.outputCommands)
 
     return process
+
+
+
+def customiseForTICLv5EventContent(process):
```

Comment:
If this extra event content is tied to TICLv5, why don't we just add it with the modifier?
Configuration/ProcessModifiers/python/ticl_v5_cff.py

Reply:
now the modifier does not use this function anymore, but it's still there because we are using it for rereco